woocommerce/woocommerce
# Fix/37601 Add `aria-describedby` to Select options button

### Submission Review Guidelines:

- I have followed the [WooCommerce Contributing Guidelines](https://github.com/woocommerce/woocommerce/blob/trunk/.github/CONTRIBUTING.md) and the [WordPress Coding Standards](https://make.wordpress.org/core/handbook/best-practices/coding-standards/).
- I have checked to ensure there aren't other open [Pull Requests](https://github.com/woocommerce/woocommerce/pulls) for the same update/change.
- I have reviewed my code for [security best practices](https://developer.wordpress.org/apis/security/). 
- Following the above guidelines will result in quick merges and clear and detailed feedback when appropriate.



### Changes proposed in this Pull Request:

* This PR adds a function to get the `aria-describedby` description for the add to cart button.
* Also adds default description for the Select options button.

Closes #37601.



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Go to the Shop page.
2. Inspect the button "Select options"
3. Check the `aria-describedby` attribute is added with the value: `"This product has multiple variants. The options may be chosen on the product page"`.

![image](https://user-images.githubusercontent.com/25176325/233335354-e582d3a4-ea47-467a-8215-3bf8ff7a6a51.png)
